src/kernel/mempool_entry.h

```diff
@@ -191,6 +201,8 @@ struct NewMempoolTransactionInfo {
      */
     int64_t m_virtual_transaction_size;
     unsigned int txHeight;
+    int64_t nSizeWithAncestors;
+    CAmount nModFeesWithAncestors;
```

Comment:
These fields added in 53bbc791d47a51a7c2bbefb32835fd4001b68890 aren't necessary as you're not using them anywhere. At quick glance I don't think we should ever tell the fee estimator about *modified* fees.